player/public/services/interfaceService.js

```diff
@@ -1,4 +1,5 @@
 org.ekstep.service.content = new(org.ekstep.service.mainService.extend({
+    assessEvents: {},
```

Comment:
It should be telemetryEvents object. It will hold assessEvents as an another object in it.